README.md

```diff
@@ -1,2 +1,70 @@
-# cloudproof_cpp
-C/C++ bindings for Cosmian Cloudproof Encryption 
+# Cloudproof C/C++
+
+C/C++ bindings for [Cosmian's Cloudproof Encryption](https://docs.cosmian.com/cloudproof_encryption/use_cases_benefits/).
+
+Cloudproof Encryption secures data repositories and applications in the cloud with advanced application-level encryption and encrypted search.
+
+<!-- toc -->
+
+- [Licensing](#licensing)
+- [Cryptographic primitives](#cryptographic-primitives)
+- [Getting started](#getting-started)
+- [Versions Correspondence](#versions-correspondence)
+
+<!-- tocstop -->
+
+## Licensing
+
+The library is available under a dual licensing scheme Affero GPL/v3 and commercial. See [LICENSE.md](LICENSE.md) for details.
+
+## Cryptographic primitives
+
+The library is based on:
+
+- [CoverCrypt](https://github.com/Cosmian/cover_crypt) algorithm which allows
+creating ciphertexts for a set of attributes and issuing user keys with access
+policies over these attributes. `CoverCrypt` offers Post-Quantum resistance.
+
+- [Findex](https://github.com/Cosmian/findex) which is a cryptographic protocol designed to securely make search queries on
+an untrusted cloud server. Thanks to its encrypted indexes, large databases can
+securely be outsourced without compromising usability.
+
+## Getting started
+
+Download the prebuilt libs and header files for released version [here](https://github.com/Cosmian/cloudproof_cpp/releases).
+
+Alternatively, the latest header files and libs can be downloaded by running the following command:
+
+```bash
+./download_libs.sh
+```
+
+Code examples are available in [./examples](./examples) to get you started.
+
+Please [check the online documentation](https://docs.cosmian.com/cloudproof_encryption/use_cases_benefits/) for more details on using the CloudProof APIs.
```

Comment:
we also could add a pre-requisites section with the minimum gcc/msvc versions needed for this sources